src/gemm_v2.cpp

```diff
@@ -159,18 +185,36 @@ static GemmBackend_t enforce_gemm_backend(miopenDataType_t data_type,
     // enforce backend based on env variable
     switch(Value(MIOPEN_GEMM_ENFORCE_BACKEND{}))
     {
+    case 0: gemm_backend_env  = GemmBackend_t::miopentensile; break;
     case 1: gemm_backend_env  = GemmBackend_t::rocblas; break;
     case 2: gemm_backend_env  = GemmBackend_t::miopengemm; break;
     case 3: gemm_backend_env  = GemmBackend_t::nogemmbackend; break;
     default: gemm_backend_env = gemm_backend_preferred;
     }
 
 // make sure backend chosen based on env variable is suppported
-#if MIOPEN_USE_ROCBLAS and MIOPEN_USE_MIOPENGEMM
+#if MIOPEN_USE_MIOPENTENSILE
     switch(gemm_backend_env)
     {
     case GemmBackend_t::nogemmbackend: gemm_backend_enforced = GemmBackend_t::nogemmbackend; break;
-    case GemmBackend_t::rocblas: gemm_backend_enforced       = GemmBackend_t::rocblas; break;
+    case GemmBackend_t::rocblas:
+    case GemmBackend_t::miopengemm:
+    case GemmBackend_t::miopentensile:
+        gemm_backend_enforced = (data_type == miopenFloat) ? GemmBackend_t::miopentensile :
+#if MIOPEN_USE_ROCBLAS
```

Comment:
https://github.com/ROCmSoftwarePlatform/MIOpen/pull/139#discussion_r478724264

It will fall back in rocblas.